bin/yb-ctl

```diff
@@ -736,18 +749,18 @@ class ClusterControl:
         for cmd_name, help in (
                 ("create", "Create a new cluster"),
                 ("start", "Create a new cluster, or start existing cluster if it already exists."),
-                ("stop", "Stops the cluster"),
-                ("destroy", "Destroy the current cluster"),
-                ("restart", "Restart the current cluster all at once"),
-                ("wipe_restart", "Stop the cluster, wipe all data files and start the cluster as"
+                ("stop", "Stop the cluster"),
+                ("destroy", "Destroy the cluster"),
+                ("restart", "Restart the cluster"),
+                ("wipe_restart", "Stop the cluster, wipe all data files and start the cluster as "
                                  "before. Will lose all the flags though."),
-                ("add_node", "Add a new node to the current cluster"),
-                ("remove_node", "Stop a particular node in the cluster."),
-                ("start_node", "Start a particular node with flags."),
+                ("add_node", "Add a new node to the cluster"),
+                ("remove_node", "Stop the specified node in the cluster"),
```

Comment:
It is a bit confusing that the command is called "remove" but the description says "stop". How is this different from stop_node?

Reply:
It's not different right now. Both just stop the node. Eventually, we should support removing a node from the cluster altogether or then drop this option.